Source/Particles/ElementaryProcess/QEDInternals/BreitWheelerEngineWrapper.cpp

```diff
@@ -123,8 +123,8 @@ BreitWheelerEngine::get_default_ctrl() const
 {
     namespace pxr_bw = picsar::multi_physics::phys::breit_wheeler;
     return PicsarBreitWheelerCtrl{
-        pxr_bw::default_dndt_lookup_table_params<amrex::Real>,
-        pxr_bw::default_pair_prod_lookup_table_params<amrex::Real>
+        pxr_bw::default_dndt_lookup_table_params<amrex::ParticleReal>,
+        pxr_bw::default_pair_prod_lookup_table_params<amrex::ParticleReal>
```

Comment:
```suggestion
        pxr_bw::default_dndt_lookup_table_params<amrex::Real>,
        pxr_bw::default_pair_prod_lookup_table_params<amrex::Real>
```

Keep these as reals?